node/.prettierignore

```diff
@@ -0,0 +1 @@
+rust-client
```

Comment:
suppress linter processing js files in rust crates
This never happes in CI, because lint check lives in a different CI task, but very helpful for a developer